pkgs/by-name/gi/gitlab-release-cli/package.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, fetchFromGitLab
```

Comment:
Needed to skip the failing test (see suggestion below)
```suggestion
, stdenv
, fetchFromGitLab
```

Reply:
Thanks for testing on darwin